packages/react-i18n/src/i18n.ts

```diff
@@ -231,6 +254,47 @@ export class I18n {
     return this.formatNumber(amount, {as: 'currency', ...options});
   }
 
+  formatCurrencyExplicit(
+    amount: number,
+    options: Intl.NumberFormatOptions = {},
+  ): string {
+    const value = this.formatCurrencyShort(amount, options);
+    const isoCode = options.currency || this.defaultCurrency || '';
+    if (value.includes(isoCode)) {
+      return value;
+    }
+    return `${value} ${isoCode}`;
+  }
+
+  formatCurrencyShort(
+    amount: number,
+    options: NumberFormatOptions = {},
+  ): string {
+    const {locale} = this;
+    const sym = this.getShortCurrencySymbol(options.currency);
+    let adjustedPrecision = options.precision;
+    if (adjustedPrecision === undefined) {
```

Comment:
I think we can clean this up a bit to prevent the use of let. We always want to use `const` whenever possible. 

Haven't tested this out but something along the lines of 

```
const currency = options.currency || this.defaultCurrency || '';
const currencyPosition = currency in CURRENCY_DIGITS ? CURRENCY_DIGITS[currency] : 2;
const adjustedPrecision = options.precision || currencyPosition;
```

Reply:
You shouldn't use `||` here because `0` is falsy and there are lots of zeros in the equation.

Reply:
^ ahh, good point.